avaframe/com1DFAPy/DFAfunctionsCython.pyx

```diff
@@ -259,7 +260,7 @@ def computeForceC(cfg, particles, fields, dem, dT):
       accNormCurv = (ux*(nxEnd-nx) + uy*(nyEnd-ny) + uz*(nzEnd-nz)) / dt
       # normal component of the acceleration of gravity
       gravAccNorm = - gravAcc * nzAvg
-      effAccNorm = gravAccNorm + accNormCurv
+      effAccNorm = gravAccNorm + curvAcceleration * accNormCurv
```

Comment:
maybe add a comment here that this is for development purposes? or that it is to turn off curvature?